pkg/plugins/bootstrap/k8s/plugin.go

```diff
@@ -86,6 +87,9 @@ func (p *plugin) BeforeBootstrap(b *core_runtime.Builder, cfg core_plugins.Plugi
 		b.WithExtensions(k8s_extensions.NewResourceConverterContext(b.Extensions(), k8s.NewSimpleConverter()))
 	}
 	b.WithExtensions(k8s_extensions.NewCompositeValidatorContext(b.Extensions(), &k8s_common.CompositeValidator{}))
+	b.WithHashing(multitenant.DefaultHashing{})
+	b.WithConfigCustomization(multitenant.DefaultPgxConfigCustomization{})
+	b.WithTenant(multitenant.DefaultTenant{})
```

Comment:
Should we do `WithMultitenancy(tenant, hashing)` as these are very likely to be always set together?